Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/syndie_business.rsi/meta.json

```
@@ -0,0 +1,30 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by ProbablyAta for use in Harmony.",
```

You should specify which platform your account is on to help alleviate ambiguity.
```suggestion
    "copyright": "Made by ProbablyAta (GitHub) for use in Harmony.",
```